src/commands/node.mjs

```diff
@@ -526,7 +526,33 @@ export class NodeCommand extends BaseCommand {
         title: 'Fetch platform software into network nodes',
         task:
           async (ctx, task) => {
-            return self.fetchPlatformSoftware(ctx, task, self.platformInstaller)
+            if (argv[flags.localBuildPath.name] !== '') {
+              const config = ctx.config
+              self.logger.debug('no need to fetch, use local build jar files')
+              const localDataLibBuildPath = argv[flags.localBuildPath.name] + '/hedera-node/data'
+              // if the path does not exist, throw an error
+              if (!fs.existsSync(localDataLibBuildPath)) {
+                throw new FullstackTestingError(`local build path does not exist: ${localDataLibBuildPath}`)
+              }
+              const subTasks = []
+              for (const nodeId of config.nodeIds) {
+                const podName = config.podNames[nodeId]
+                subTasks.push({
+                  title: `Copy local build to Node: ${chalk.yellow(nodeId)}`,
+                  task: async () => {
+                    this.logger.debug(`Copying build files to pod: ${podName}`)
+                    await self.k8.copyTo(podName, constants.ROOT_CONTAINER, localDataLibBuildPath, `${constants.HEDERA_HAPI_PATH}`)
```

Comment:
Please note `k8.copyTo` was found to be a bit unstable for large files as in this case the `build.zip`. However with latest releases of the Kubeclient it might have been already fixed. Here is a PR where I stopped using it in our e2e test: https://github.com/hashgraph/solo/pull/174

Reply:
So if we encounter issue, we need to go back and ensure `k8.copyTo` is stable with large files (probably just add a new e2e test that uploads a large file ~100Mb

Reply:
@matteriben did some research and I think he made notes in our issue tracking the bug.